### ISSUE TITLE
GraphQL Get `group` filter missing

### ISSUE THREAD

**Issue by laura-ham** [semi-technologies/weaviate-javascript-client#7]
The GraphQL query with `group` filter for entity merging is missing. Queries like this are impossible with the client now:

```graphql
{
  Get {
    Things {
      Publication(
        group:{
          type: merge,
          force:0.05
        }
      ) {
        name
      }
    }
  }
}
```

**Comment by etiennedi** [comment]
Should be fixed/implemented in `v1.0.0-rc4`. Keeping this issues open until I have feedback that it works as desired.

**Comment by etiennedi** [comment]
Confirmed working by @laura-ham. Closing.
